pages/community.tsx

```diff
@@ -0,0 +1,397 @@
+      },
+    ],
+  },
+];
+
+const galleryImages = [
+  {
+    src: "/assets/community/gallery/1.png",
+  },
+  {
+    src: "/assets/community/gallery/2.png",
+  },
+  {
+    src: "/assets/community/gallery/3.png",
+  },
+  {
+    src: "/assets/community/gallery/4.png",
+  },
+  {
+    src: "/assets/community/gallery/5.png",
+  },
+  {
+    src: "/assets/community/gallery/6.png",
+  },
+  {
+    src: "/assets/community/gallery/7.png",
+  },
+  {
+    src: "/assets/community/gallery/8.png",
+  },
+  {
+    src: "/assets/community/gallery/9.png",
+  },
+];
+
+const Community: ThirdwebNextPage = () => {
+  const trackEvent = useTrack();
+
+  return (
+    <DarkMode>
+      <NextSeo {...SEO} />
+      <Flex
+        sx={{
+          // overwrite the theme colors because the home page is *always* in "dark mode"
+          "--chakra-colors-heading": "#F2F2F7",
+          "--chakra-colors-paragraph": "#AEAEB2",
+          "--chakra-colors-borderColor": "rgba(255,255,255,0.1)",
+          "--product-accent-color": "rgba(24, 67, 78, 0.8)",
+        }}
+        justify="center"
+        flexDir="column"
+        as="main"
+        bg="#000"
+      >
+        <HomepageTopNav />
+        {/* pull it up by as much as the topnav is tall */}
+        <Box mt="-80px" overflowX="hidden">
+          <Center
+            py={{ base: 12, md: 24 }}
+            px={{ base: 4, md: 8 }}
+            flexDir="column"
+          >
+            <Heading mt={8} textAlign="center" fontSize="64px">
+              A decentralized internet begins
+              <br /> with{" "}
+              <Text
+                as="span"
+                fontSize="48px"
+                fontWeight="bold"
+                background="linear-gradient(243.9deg, #BFA3DA 21.81%, #84309C 48.81%, #C735B0 86.61%), #ECECEC"
+                backgroundClip="text"
+                color="transparent"
+              >
+                decentralized talent.
+              </Text>
+            </Heading>
+
+            <LinkButton
+              href="https://discord.gg/thirdweb"
+              size="lg"
+              onClick={() =>
+                trackEvent({
+                  category: TRACKING_CATEGORY,
+                  action: "click",
+                  title: "Join the community",
+                })
+              }
+              background="white"
+              _hover={{
+                background: "white",
+                color: "#000",
+              }}
+              color="#000"
+              fontSize="18px"
+              leftIcon={<Icon as={FiZap} />}
+              mt={8}
+            >
+              Join the community
+            </LinkButton>
+          </Center>
+
+          {CommunitySections.map(
+            ({ title1, title2, cards, description }, i) => (
+              <Box
+                w="100%"
+                as="section"
+                zIndex={2}
+                position="relative"
+                key={title1}
+              >
+                <Box
```

Comment:
why do we need this box?

Reply:
this is a divider between each section. Should I remove it?